mitodl/release-script
# Add PR karma

#### What are the relevant tickets?
Fixes #72 

#### What's this PR do?
 - Adds a command to calculate pull request karma. Example usage: `karma 2017-11-09` to calculate karma between that date and today.
 - Adds `bot_local.py` which runs a single command on a particular channel, but without actually connecting to slack.

#### How should this be manually tested?
 - Get a github access token. You can create one in the settings for your account. Use it for `GITHUB_ACCESS_TOKEN`
 - Run `./bot_local.py karma 2017-11-09`. You should see a list of people and their karma score, sorted from biggest to smallest.
